src/_shared/js/capi-multiple.js

```diff
@@ -137,29 +134,32 @@ function editionLink (host, edition, adType) {
 }
 
 // Uses cAPI data to build the ad content.
-function buildFromCapi (host, cardsInfo, adType, modifyCardFn) {
+function buildFromCapi (host, cardsInfo, adType, generateLogo) {
 
     let cardList = document.createDocumentFragment();
 
     cardsInfo.isSingle = cardsInfo.articles
-    .map(cardInfo => cardInfo.branding.sponsorLogo.url)
-    .reduce(((isSingle, url, index, urls) => isSingle && (index === 0 || url === urls[index - 1])), true);
+        .map(cardInfo => cardInfo.branding && cardInfo.branding.logo.src)
+        .reduce(((isSingle, url, index, urls) => isSingle && (index === 0 || url === urls[index - 1])), true);
 
     // Constructs an array of cards from an array of data.
     cardsInfo.articles.forEach((info, idx) => {
-        cardList.appendChild(buildCard(info, idx, adType, cardsInfo, modifyCardFn));
+        cardList.appendChild(buildCard(info, idx, adType, cardsInfo, generateLogo));
     });
 
     return write(() => {
-        // Takes branding from last possible card, in case earlier ones overriden.
-        if( cardsInfo.isSingle ) addBranding(cardsInfo.articles.slice(-1)[0]);
+        // Takes branding from last possible card, in case earlier ones overridden.
+        var brandingCard = cardsInfo.articles.slice(-1)[0];
+        if( cardsInfo.isSingle ) {
+            addBranding(brandingCard, generateLogo);
```

Comment:
this would be more problematic. perhaps this `customise` function should be called with the whole fragment instead...